src/protocol/libp2p/bitswap/handle.rs

```diff
@@ -107,6 +109,13 @@ impl BitswapHandle {
     pub async fn send_response(&self, peer: PeerId, responses: Vec<ResponseType>) {
         let _ = self.cmd_tx.send(BitswapCommand::SendResponse { peer, responses }).await;
     }
+
+    #[cfg(feature = "fuzz")]
+    /// Expose functionality for fuzzing
+    pub async fn fuzz_send_message(&mut self, command: BitswapCommand) -> crate::Result<()> {
```

Comment:
nit: Since the `BitswapCommand` has all the fields exposed, could we instead use `send_response`? 🤔